README.md

```diff
@@ -10,14 +10,6 @@ Ensure your finished repository meets all of the [Assessment Criteria](#assessme
 
 ## Assessment Criteria
 
-1. The root URL should serve/return the file `index.html`. The `index.html` file should outline the expected endpoints for the service.
-
-1. The `index.html` file should reference a `favicon.ico`, `logo.png` and a `styles.css` file.
-
-1. Add a `<title>` tag that includes your name e.g. Welcome to `Monica's` Todo List API
-
-1. Include the table of endpoints below in your homepage (`index.html`) as a description of how to consume (use) your API.
-
```

Comment:
Do the students no longer need to do any client side work for this assessment?

Reply:
@tanyapowell Yes, the HTML task feels like an unnecessary distraction. Most questions from current cohort are about how to convert the Markdown table into HTML and we're not testing anything but the response code, so seemed wisest to let them focus on actual Node work?